scripts/create_manifest_docker.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+set -e
+source "$(dirname $(realpath $0))/common.sh"
+
+main() {
+  ecr_login
+
+  docker manifest create --amend $ECR_URL/$IMAGE \
```

Comment:
If I read this correctly then this will expand to `$ECR_URL/aws-nitro-enclaves-k8s-device-plugin:v0.2-x86_64` when run on an x86 machine. I do not think this is what we actually want. (?)

Probably better to stay with `$ECR_URL/$REPOSITORY_NAME` as it was before.

Reply:
`arch` got removed from image tag. Image just points at the generic image name + version tag now: https://github.com/aws/aws-nitro-enclaves-k8s-device-plugin/pull/20/files#diff-b23204e92f793e233cf28605758e6f1d4849830577492c85f00593f977527e27R17